packages/client/package.json

```diff
@@ -57,13 +57,14 @@
     "workspace:dev": "cross-env TARGET=workspace vite --config vite-ws.config.ts",
     "workspace:build": "cross-env TARGET=workspace vite build --config vite-ws.config.ts",
     "workspace:serve": "cross-env TARGET=workspace vite serve --config vite-ws.config.ts",
-    "workspace:run": "cross-env-shell \"wait-on -l http://localhost:2017/config/workspace.json && openfin -l -c http://localhost:2017/config/workspace.json\"",
+    "workspace:run": "cross-env-shell \"wait-on -l http://localhost:2017/config/workspace.json && openfin -l http://localhost:2017/config/workspace.json\"",
```

Comment:
watch out for consistency .. it is unchanged on L42 .. I am updating all of this in another branch, so dw for now